src/basicfuns.jl

```diff
@@ -4,22 +4,32 @@
 """
     xlogx(x::Real)
```

Comment:
The signature still needs to be updated here.

Also below `x = 0` isn't correct for complex numbers, better say "if x is zero".

Reply:
Right, thanks again for noticing! See https://github.com/JuliaStats/StatsFuns.jl/pull/96